docusaurus/docs/React/components/message-components/render-text.mdx

```diff
@@ -79,24 +67,18 @@ const MentionComponent = ({ children, node: { mentionedUser } }) => (
   </a>
 );
 
-const App = () => (
-  <Chat client={client}>
-    <Channel>
-      <Window>
-        <MessageList
-          renderText={(text, mentionedUsers) =>
-            renderText(text, mentionedUsers, {
-              customMarkDownRenderers: { strong: StrongComponent, mention: MentionComponent },
-            })
-          }
-        />
-      </Window>
-    </Channel>
-  </Chat>
+export const WrappedMessageList = () => (
+  <MessageList
+    renderText={(text, mentionedUsers) =>
+      renderText(text, mentionedUsers, {
+        customMarkDownRenderers: { strong: StrongComponent, mention: MentionComponent },
```

Comment:
Shall we start using the `My*` prefix for the custom components? I think it makes it more clear for the reader that these components are not coming from the SDK.

Something like: `MyStrongComponent` and `MyMentionComponent`. This a practice we follow on video.

Also, it would be good if we have a simple example here:

```tsx
const MyStrongComponent = ({ text }) => <strong>{text}</strong>;
```

Reply:
I agree, that makes sense. I'll update accordingly. :)